api/RTCIceCandidate.json

```diff
@@ -133,7 +133,7 @@
               "version_added": null
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": false
             },
```

Comment:
This file they all went from true to false.

I am 99% sure that RTCIceCandidate is in the browser since it is essential for WebRTC, but should probably check.

Reply:
... This seems accurate...